nemo_rl/models/policy/workers/dtensor_policy_worker_v2.py

```diff
@@ -233,6 +265,9 @@ def __init__(
                 torch_dtype=str(model_config.torch_dtype),
             )
 
+            if self.peft_config is not None:
```

Comment:
nit: can this be made consistent with the checks below

```suggestion
            if self.lora_enabled:
```